### ISSUE TITLE
New UI lacks any way to scroll down or resize windows the second

### ISSUE THREAD

**Issue by Huenererschrecker** [VOREStation/VOREStation#9169]
#### Brief description of the bug
Same oversight as last time, only this time I've specificly noticed the lack of a scrollbar on the ID card modification console.

#### What you expected to happen
The UI to have a decent way to handle scrolling up or down in their window.

#### What actually happened
No obvious way to scroll up or down in the window. Only the mousewheel works.

#### Steps to reproduce
- Just use the ID manipation console my man.

#### Code Revision
- 7f729ff313e8b421f547ece99025629b988b90fe 
 

#### Anything else you may wish to add:
- The closed issue last time https://github.com/VOREStation/VOREStation/issues/9072

**Comment by ShadowLarkens** [comment]
This is handled on a per-UI basis, so, I'm going to need either this issue updated with every one you notice, or new issues created for each UI.

**Comment by Huenererschrecker** [comment]
The problem is that those problems are either there or not and not every menu is readily available. I can only report them when I find them, so I doubt waiting on one large one is a good idea.

**Comment by ShadowLarkens** [comment]
Mkay! Just keep making new issue reports as you find UIs that need scrolling support then, I'll try to resolve them as ya find em.

**Comment by ShadowLarkens** [comment]
See above, please make new issues for these problems with the specific UI listed ^
